Demo/SwiftUI/PaymentsViewModel.swift

```diff
@@ -21,7 +21,7 @@ internal final class PaymentsViewModel: ObservableObject, Identifiable, Presente
     // MARK: - DropIn Component
 
     internal func presentDropInComponent() {
-       // todo: add advanced flow
```

Comment:
How was it working without `dropIn.presentDropInComponent()` ?

Reply:
I had this **todo:**  in my last PR.